src/main/cpp/t06_sum.cpp

```diff
@@ -22,7 +22,24 @@
 
 using namespace std;
 
+int sum(int n, int s)
+{
+	while (n != 0) {
+
+		s = s + n;
+		cin >> n;
+		sum(n, s);
+		if (n == 0) cout << s;
+	}
+	return s;
+}
 int t06_sum() {
 
-    return 0;
+	int result, d, s;
+	cin >> d;
+	s = 0;
+	result = 0;
+	result = sum(d, s);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'result' is assigned a value that is never used.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4951807)